quickstart/setup_db.cql

```diff
@@ -1,2 +1,2 @@
 CREATE KEYSPACE IF NOT EXISTS k1 WITH replication = {'class':'SimpleStrategy', 'replication_factor':1};
-CREATE TABLE IF NOT EXISTS k1.fruit(store_id, name text, description text, PRIMARY KEY((store_id), name));
\ No newline at end of file
+CREATE TABLE IF NOT EXISTS k1.fruit(store_id text, name text, description text, PRIMARY KEY((store_id), name));
```

Comment:
This script is removed on the current `master` so please revert this change, otherwise, you'll get a confict